spec/controllers/idv/in_person/usps_locations_controller_spec.rb

```diff
@@ -283,26 +283,6 @@
       end
     end
 
-    context 'with hybrid user' do
```

Comment:
i think we had this spec b/c a hybrid user can end up in the in person flow. What was the error that was observed before removing this spec?

Reply:
The error I was getting was enrollment on line 297 would end up being nil

Reply:
That's a great question @svalexander ! The TLDR is that yes, I do believe we want to keep this test.

I had to do a lot of digging because I wasn't sure initially, and this part of the code has changed a lot in other work. In our workflow we don't end up going to the "try in person" option very often.

After that research I still wasn't sure, and I ended up testing it out locally. I found that yes - we do have the ability to search for post offices on the mobile side when in the hybrid flow, if ipp is enabled and we have an attention result. I believe that's what this is testing.

Unfortunately I'm not sure what to do about the user being nil in this code change. I think that would take me a good deal more research.

Reply:
@night-jellyfish OK it is returning nil because I removed effective_user. We removed it because hybrid flow was only pertaining to uploading a photo and viewing if that was successful. If people also can look up USPS locations on the hybrid flow I might have to add something to check for doc_capture_user_id.

Reply:
If the user is in the in-person flow, they need to be fully authenticated, right? `effective_user` was meant to support photo upload via phones only. Any other idv pages should be looking at `current_user` (unless i am missing something major about how IPP works)